roadmap.md

```diff
@@ -1,68 +1,39 @@
-    * [UIAutomationProvider](src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationProvider)
-    * [UIAutomationType](src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationTypes)
-  * [WPF Extensions](src/Microsoft.DotNet.Wpf/src/Extensions)
-    * [PresentationFramework-SystemCore](src/Microsoft.DotNet.Wpf/src/Extensions/PresentationFramework-SystemCore)
-    * [PresentationFramework-SystemData](src/Microsoft.DotNet.Wpf/src/Extensions/PresentationFramework-SystemData)
-    * [PresentationFramework-SystemDrawing](src/Microsoft.DotNet.Wpf/src/Extensions/PresentationFramework-SystemDrawing)
-    * [PresentationFramework-SystemXml](src/Microsoft.DotNet.Wpf/src/Extensions/PresentationFramework-SystemXml)
-    * [PresentationFramework-SystemXmlLinq](src/Microsoft.DotNet.Wpf/src/Extensions/PresentationFramework-SystemXmlLinq)
-  * [WindowsFormsIntegrations](src/Microsoft.DotNet.Wpf/src/WindowsFormsIntegration)
-  * [System.Windows.Controls.Ribbon](src/Microsoft.DotNet.Wpf/src/System.Windows.Controls.Ribbon)
-  * [WPF Themes](src/Microsoft.DotNet.Wpf/src/Themes)
-    * [PresentationFramework-Aero](src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Aero)
-    * [PresentationFramework-Aero2](src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Aero2)
-    * [PresentationFramework-AeroLite](src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.AeroLite)
-    * [PresentationFramework-Classic](src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Classic)
-    * [PresentationFramework-Luna](src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Luna)
-    * [PresentationFramework-Royale](src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Royale)
-  * [System.Windows.Presentation](src/Microsoft.DotNet.Wpf/src/System.Windows.Presentation)
-  * [PresentationUI](src/Microsoft.DotNet.Wpf/src/PresentationUI)
-  * [System.Printing](src/Microsoft.DotNet.Wpf/src/System.Printing)
-  * Native Components
-    * [PenIMC_Cor3](src/Microsoft.DotNet.Wpf/src/PenImc)
-    * [WpfGfx_Cor3](src/Microsoft.DotNet.Wpf/src/WpfGfx)
-  
-* Tests:
-  * [DrtXaml](src/Microsoft.DotNet.Wpf/test/DRT/DrtXaml)
-
+# WPF for .NET 5 Roadmap
+WPF is a .NET Core UI framework for building Windows desktop applications. The ownership of WPF was recently transitioned to our team (Developer Ecosystem and Platform team under Windows).This transition enables investments across UI frameworks namely WINUI and WPF to stay aligned and remain future-proof as new technology trends & devices are introduced in the industry.
+
+The roadmap below communicates priorities for evolving and extending the scope of WPF for .NET Core through 2020 and into 2021. It will continue to evolve based on market changes and customer feedback, so please note that the plans outlined here are not exhaustive or guaranteed. We welcome your feedback on the roadmap: please feel free to contribute to existing issues or [file a new issue](https://github.com/dotnet/wpf/issues "file a new issue").
```

Comment:
```suggestion
The roadmap below communicates priorities for evolving and extending the scope of WPF for .NET Core through 2020 and into 2021. It will continue to evolve based on market changes and customer feedback, so please note that the plans outlined here are not exhaustive or guaranteed. We welcome your feedback on the roadmap: please feel free to contribute to existing issues or [file a new issue](https://github.com/dotnet/wpf/issues/new/choose).
```
corrected link